### PR TITLE
Improve developer documentation

### LINKED ISSUE #1019: Add documentation for developers using bitbucket
We should add documentation for developers that use bitbucket, since integration of SLDS is different.

Some tips from slack:

1. The slds token should be available within the Workspace variables pipeline on bitbucket
2. You should use that variable within the `.npmrc` file

```
echo "@sl-design-system:registry=https://npm.pkg.github.com/" >> ${BITBUCKET_CLONE_DIR}/frontend/.npmrc
echo "//npm.pkg.github.com/:_authToken=$SLDESIGNSYSTEMS_GITHUB_NPM_AUTH_TOKEN" >> ${BITBUCKET_CLONE_DIR}/frontend/.npmrc
```

Keep in mind that slds prerequisition is to have font awesome included in .npmrc, withi etsl it looks like:

```
echo "@awesome.me:registry=https://npm.fontawesome.com/" >> ${BITBUCKET_CLONE_DIR}/frontend/.npmrc
echo "@fortawesome:registry=https://npm.fontawesome.com/" >> ${BITBUCKET_CLONE_DIR}/frontend/.npmrc
echo "//npm.fontawesome.com/:_authToken=$NPM_FA_TOKEN" >> ${BITBUCKET_CLONE_DIR}/frontend/.npmrc
```
where `NPM_FA_TOKEN` is defined within our repository variable

```[tasklist]
### Tasks
- [ ] Find someone who has experience with bitbucket
- [ ] add documentation on documentation website on getting started for developers in FAQ section
```

### LINKED ISSUE #1057: Improve getting started for developers
```[tasklist]
### Tasks
- [ ] document that polyfills are needed
- [ ] make a checklist at the top of that page that devs can easily follow
- [ ] mention peer dependencies
- [ ] describe easy check to see if a web component is loaded (#shadow-root in devtools)
```

### PR DESCRIPTION
**Pull request by jpzwarte**
- Fixes #1019
- Fixes #1057

### CODE REVIEW

File: website/src/categories/getting-started/developers.md [hunk 1]
```diff
@@ -60,36 +77,34 @@ To get started using the SL Design System, you can install it via npm. Each comp
 
 ## Setup a theme
 
-To start using the SL Design System, you need to setup a theme for your application. The theme determines how your application looks. There is a theme for every major product within Sanoma Learning. You can install a theme as by installing the NPM package, in this example we use the `sanoma-learning` theme.
+To start using the SL Design System, you need to setup a theme for your application. The theme determines how your application looks. There is a theme for every major product within Sanoma Learning.
 
-<div class="ds-code">
+Please note that the webfonts of your application are not a part of the theme. There are different ways of loading the fonts an application used across the products in Sanoma Learning. If is up to you to load the webfonts in your application.
```
Comment (Diaan) [inline website/src/categories/getting-started/developers.md:82]: There is a section about fonts a bit further down, maybe it is better to add this info there?

File: website/src/categories/getting-started/developers.md [hunk 2]
```diff
@@ -60,36 +77,34 @@ To get started using the SL Design System, you can install it via npm. Each comp
...
-
-After installing the theme, you need to import the theme in your application. If you are working in an Angular application you can do this by loading the theme in your application in your angular.json, in all the places where you include your other css files:
+You can install a theme as by installing the NPM package, in this example we use the `sanoma-learning` theme.
 
 <div class="ds-code">
 
-  ```json
-   './node_modules/@sl-design-system/sanoma-learning/all.css'
+  ```bash
+  npm add @sl-design-system/sanoma-learning
```
Comment (Diaan) [inline website/src/categories/getting-started/developers.md:89]: in other places we also offer the yarn version.

(this part can eventually be replaced by the new component i made)

**Review by Diaan — APPROVED** [review]

File: website/src/categories/getting-started/developers.md [hunk 3]
```diff
@@ -20,12 +20,25 @@ eleventyNavigation:
 
 <section>
 
+## Steps
+
+Please follow the steps below when first getting started:
+1. [Make sure you have access](#make-sure-you-have-access)
+2. [Installing the SL Design System package(s)](#installing-the-sl-design-system-packages)
```
Comment (anna-lach) [inline website/src/categories/getting-started/developers.md:27]:
```suggestion
2. [Installing the SL Design System package(s)](#installing-the-sl-design-system-package(s))
```

File: website/src/categories/getting-started/developers.md [hunk 4]
```diff
@@ -44,12 +57,16 @@ Make sure you add the token to `~/.npmrc`, the file in your home directory, not
...
+
+Some packages have 3rd party dependencies that need to be installed as well. Some of these dependencies may already be used in your own application. To make sure that those dependencies are not installed twice, they are marked as `peerDependencies` in the `package.json` of the SLDS packages. This means that you need to install them yourself. For example, the `@sl-design-system/badge` package has `lit` as a peer dependency. If you are using `lit` in your application, you don't need to install it again. If you are not using `lit` yet, you need to install it. Your package manager will warn you if you haven't installed these peer dependencies yet.
+
+So if you want to use a `button` component with the Sanoma Learning theme, you do:
 
 <div class="ds-code">
 
   ```bash
-  npm install @sl-design-system/button 
+  npm add @sl-design-system/button
```
Comment (anna-lach) [inline website/src/categories/getting-started/developers.md:69]: npm / yarn

File: website/src/categories/getting-started/developers.md [hunk 5]
```diff
@@ -160,13 +198,25 @@ Because the font name in the token needs to match the name of the font used in F
...
 
+### How do I setup my Bitbucket pipeline to work with the SLDS packages?
+
+For the Sanoma Learning Bitbucket pipelines, there is a common token that you can use to authenticate with the GitHub NPM registry. For your projects' `.npmrc` you can use the following line to authenticate with the GitHub NPM registry:
+
+```
+//npm.pkg.github.com/:_authToken=${SLDESIGNSYSTEMS_GITHUB_NPM_AUTH_TOKEN}
+```
+
 ### Which versions of Angular are supported?
```
Comment (anna-lach) [inline website/src/categories/getting-started/developers.md:220]: Maybe we should also add a question `What browsers are supported`?

File: website/src/categories/getting-started/developers.md (hunk 1 above)
Comment (jpzwarte) [reply website/src/categories/getting-started/developers.md]: The FAQ item is more about the fonts not working, even though you have loaded them. So I'll leave this bit here.

**Pull request merged**